[INFO -- 2017-03-04 03:34:34]
Starting flip_value script
version: 2.0
[INFO -- 2017-03-04 03:34:34]
Initialization strings:
[INFO -- 2017-03-04 03:34:34]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-03-04 03:34:34]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-04 03:34:34]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-04 03:34:34]
(4) file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-04 03:34:34]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-04 03:34:34]
(6) set args 228 2048 16 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_a_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_b_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/gold_2048_m-order_228_ths_16_blocks 1
[INFO -- 2017-03-04 03:34:34]
initSignal:28
[INFO -- 2017-03-04 03:34:34]
endSignal:28
[INFO -- 2017-03-04 03:34:34]
seqSignal:15
[INFO -- 2017-03-04 03:34:34]
faultMode:3
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[EXCEPTION -- 2017-03-04 03:35:17]
pythonException: invalid frame
[INFO -- 2017-03-04 03:35:17]
Backtrace BEGIN:
#0  0x00000000004038fc in L_dgemm_185__par_region0_2_0 () at dgemm.c:224
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffe550 in ?? ()
#3  0x0000000000000000 in ?? ()

219	                        for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
220	                            for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
221	                                CC_arr(i,j) += AA_arr(i,k)*BB_arr(j,k);
222	
223	                    for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
224	                        for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
225	                            C_arr(ig,jg) += CC_arr(i,j);
226	
227	                }
228	            }

Backtrace END
Fliping a bit of the value pointed by a pointer
Memory content before bitflip:0x160x6f0x6a0x790xe90x420x500xc3
Fault Mode: Zero bit-flip
base address to flip value: 0x7ffff2ccb040
address max offset: 8
Memory content after  bitflip:0x000x000x000x000x000x000x000x00
frame name: L_dgemm_185__par_region0_2_0
symbol name: B
symbol filename: dgemm.c
symbol line: 185
value: 0x7ffff2ccb040
value address: 0x7fffffffe548
Type: The type is a pointer.
Type sizeof: 8

[INFO -- 2017-03-04 03:35:17]
Fault Injection Successful after 43.308576107s
[INFO -- 2017-03-04 03:35:44]
event type: exit
[INFO -- 2017-03-04 03:35:44]
exit code: 0
